web/modules/custom/portland/portland.module

```diff
@@ -813,123 +803,115 @@ function portland_cron(){
     $reviewers = $node->field_reviewer;
-  }
-
-  $revision_log_output = $node->revision_log->value ? 
-      "<div><h3>Revision log:</h3>\r\n".$node->revision_log->value."</div>\r\n<p/>\r\n<div>" : '';
-
-  $moderation_state_message = [
-    'review' => 'In review',
-    'draft' => 'Draft',
-    'archived' => 'Unpublished/Archived',
-    'published' => 'Published',
-  ];
-
-  // send email
-  $key = 'portland_notify_reviewers';
-  $params['title'] = 'Portland.gov reviewer update';
-  $langcode = \Drupal::currentUser()->getPreferredLangcode();
-  $params['message'] = "You have been assigned to review <a href=\"/node/".$node->id().
-    "/revisions/".$node->vid->value."/view\"> ".$latest_rev->label().
-    "</a> . You will be notified of all changes to this content.<p/>". 
-    "<a href=\"".$node->revision_uid->entity->url()."\">".$node->revision_uid->entity->label().
-    "</a> ".$status_message." a new <strong>".$moderation_state_message[$latest_rev->moderation_state->value].
-    "</strong> revision at ".
-    \Drupal::service('date.formatter')->format($node->changed->value, 'custom', 'l Y/n/j g:i A').
-    ".<p/>".$revision_log_output;
-
-  if($reviewers == NULL) return;
-  $current_user_email = \Drupal::currentUser()->getAccount()->getEmail();
-  foreach ($reviewers->referencedEntities() as $reviewer) {
-    $reviewer_email = $reviewer->mail->value;
-    _send_email($key, $reviewer_email, $langcode, $params, $current_user_email);
-  }
+    
+    $revision_log_output = $node->revision_log->value ? 
+        "<div><h3>Revision log:</h3>\r\n".$node->revision_log->value."</div>\r\n<p/>\r\n<div>" : '';
+    
+    $moderation_state_message = [
+        'review' => 'In review',
+        'draft' => 'Draft',
+        'archived' => 'Unpublished/Archived',
+        'published' => 'Published',
+    ];
+    
+    // send email for any moderation state except draft
+    if ($latest_rev->moderation_state->value != 'draft') {
+        $key = 'portland_notify_reviewers';
+        $params['title'] = 'Portland.gov reviewer update';
+        $langcode = \Drupal::currentUser()->getPreferredLangcode();
+        $params['message'] = "You have been assigned to review <a href=\"/node/".$node->id().
+            "/revisions/".$node->vid->value."/view\"> ".$latest_rev->label().
+            "</a> . You will be notified of all changes to this content.<p/>". 
+            "<a href=\"".$node->revision_uid->entity->url()."\">".$node->revision_uid->entity->label().
+            "</a> ".$status_message." a new <strong>".$moderation_state_message[$latest_rev->moderation_state->value].
+            "</strong> revision at ".
+            \Drupal::service('date.formatter')->format($node->changed->value, 'custom', 'l Y/n/j g:i A').
+            ".<p/>".$revision_log_output;
+        
+        if ($reviewers == NULL) return;
+        $current_user_email = \Drupal::currentUser()->getAccount()->getEmail();
+        foreach ($reviewers->referencedEntities() as $reviewer) {
```

Comment:
Could we add a conditional check here to skip the current revision editor from being notified?

Reply:
I suppose so, if you want to expand the scope of the story to include this feature.

Reply:
Nice!